requirements.txt

```diff
@@ -0,0 +1 @@
+praw=4.4.0
```

Comment:
Not sure how I feel about the mismatch between this file and the dependencies declared in setup.py. I'd almost get rid of the requirements.txt file since we have a fully-fledged pip install manifest.